vrf-listener/tests/integration/vrf_listener_test.py

```diff
@@ -152,12 +146,9 @@ async def test_vrf_listener_miss_with_large_block(
 
```

Comment:
We can remove this `test_vrf_listener_miss_with_large_block` test since block isn't a parameter anymore